fission/src/mirabuf/MirabufInstance.ts

```diff
@@ -126,39 +125,35 @@ class MirabufInstance {
     }
 
     /**
-     * Parses all mirabuf appearances into ThreeJs materials.
+     * Parses all mirabuf appearances into ThreeJS and Jolt materials.
      */
     private LoadMaterials(materialStyle: MaterialStyle) {
+        Object.entries(this._mirabufParser.assembly.data!.materials!.physicalMaterials!).forEach(([name, material]) => {
+            const [static_f, dynamic_f] = [material.staticFriction, material.dynamicFriction]
+            console.log(`${name} - static: ${static_f} dynamic: ${dynamic_f}`)
+            //this._materials.set()
+        })
         Object.entries(this._mirabufParser.assembly.data!.materials!.appearances!).forEach(
             ([appearanceId, appearance]) => {
-                let hex = 0xe32b50
-                let opacity = 1.0
-                if (appearance.albedo) {
-                    const { A, B, G, R } = appearance.albedo
-                    if (A && B && G && R) {
-                        hex = (A << 24) | (R << 16) | (G << 8) | B
-                        opacity = A / 255.0
-                    }
-                }
-
-                let material: THREE.Material
-                if (materialStyle == MaterialStyle.Regular) {
-                    material = new THREE.MeshPhongMaterial({
-                        color: hex,
-                        shininess: 0.0,
-                        shadowSide: THREE.DoubleSide,
-                        opacity: opacity,
-                        transparent: opacity < 1.0,
-                    })
-                } else if (materialStyle == MaterialStyle.Normals) {
-                    material = new THREE.MeshNormalMaterial()
-                } else if (materialStyle == MaterialStyle.Toon) {
-                    material = World.SceneRenderer.CreateToonMaterial(hex, 5)
-                    console.debug("Toon Material")
-                }
-
-                World.SceneRenderer.SetupMaterial(material!)
-                this._materials.set(appearanceId, material!)
+                const { A, B, G, R } = appearance.albedo ?? {}
+                const [hex, opacity] =
+                    A && B && G && R ? [(A << 24) | (R << 16) | (G << 8) | B, A / 255.0] : [0xe32b50, 1.0]
```

Comment:
Since we run out of line real estate here anyways I think we should move these onto two separate lines.